src/controller_examples/rabbitmq_controller/proof/liveness/proof.rs

```diff
@@ -28,6 +28,63 @@ use vstd::{prelude::*, string::*};
 
 verus! {
 
+// We prove init /\ []next /\ []wf |= []RMQCluster::desired_state_is(rabbitmq) ~> []current_state_matches(rabbitmq) holds for each rabbitmq.
+proof fn liveness_proof_forall_rabbitmq()
+    ensures
+        forall |rabbitmq: RabbitmqClusterView| rabbitmq.well_formed() ==> #[trigger] cluster_spec().entails(liveness(rabbitmq)),
+{
+    assert forall |rabbitmq: RabbitmqClusterView| rabbitmq.well_formed()
+    implies #[trigger] cluster_spec().entails(liveness(rabbitmq)) by {
+        liveness_proof(rabbitmq);
+    };
+}
+
+proof fn liveness_proof(rabbitmq: RabbitmqClusterView)
+    requires
+        rabbitmq.well_formed(),
```

Comment:
Actually, do we really need `rabbitmq.well_formed()` throughout the proof since the reconcile_core spec always assumes rabbitmq is well formed?

Reply:
Of course I might miss something here

Reply:
I am asking this coz I am thinking about removing the well formed thing from the top level liveness property, unless it will break the current proof.